docs/troubleshooting.md

```diff
@@ -101,3 +102,44 @@ Don’t use distinct() as a metric with a large amount of data if you are using
 ### Heap memory size
 The number of rows IQL can return on non-streaming queries depends on the heap size allocated to IQL.
 
+## Performance Considerations for IQL Usage
+
+### Query smaller time ranges
+First test your query on a tiny time range and then ramp up to the required range if performance is sufficient. 
+E.g. `FROM example 1h today ...` is a better way to test than `FROM example 180d today ...`.
+
+If you have created a Superset or Aquarium Dashboard, consider the time range. Do you really need to see 90d today worth of data, or is 2w today sufficient?
```

Comment:
These references are to Indeed-internal tools and need to be removed. I think this whole paragraph can go.

Reply:
@hnarasaki actually already removed this line in pull #34, which was also merged